cli/pack/templates/Dockerfile.pack.build

```diff
@@ -0,0 +1,45 @@
+# Install tt
+FROM golang:1.18 AS tt-builder
+
+WORKDIR /usr/src/mage
+
+RUN apt update && apt install -y patch git
+
+RUN git clone https://github.com/magefile/mage && \
+    cd mage && \
+    go run bootstrap.go
+
+WORKDIR /usr/src/tt
+
+RUN git clone https://github.com/tarantool/tt.git --recursive
```

Comment:
I'm not sure this is good to always build tt from master branch. If we are not going to repeat tt host version, it is faster to install tt from repository.

Reply:
You mean to use package managers?

Reply:
Yes. I think it is safer to install the latest tt version from a repo, than building it from master branch.

Reply:
Fixed.